package.json

```diff
@@ -5,7 +5,8 @@
   "keywords"      : ["model", "view", "controller", "router", "server", "client", "browser"],
   "author"        : "Jeremy Ashkenas <jeremy@documentcloud.org>",
   "dependencies"  : {
-    "underscore"  : ">=1.6.0"
+    "underscore"  : ">=1.6.0",
+    "jquery"      : ">=2.0.0"
```

Comment:
This should be like `1.10.x - 2.x` or w.e., right? That way your app can determine which version of jQuery to use.